appendices/editor-in-chief-checks.md

```diff
@@ -29,10 +29,12 @@ NOTE: We prefer that you have development instructions in your documentation too
 - [ ] **Archive** (JOSS only, may be post-review): The repository DOI resolves correctly.
 - [ ] **Version** (JOSS only, may be post-review): Does the release version given match the GitHub release (v1.0.0)?
 
+**Optional:** Let projects know that it's a great idea for projects to have a .github repository for the project organization where they can host a commonly used LICENSE, Code of Conduct, and even a YAML file with label definitions. These items will then be automatically applied to every repository in the organization to ensure consistency (but can be customized within repos too). The [SunPy project](https://github.com/sunpy/.github/) has a great example of this.
```

Comment:
This is the major change. The other items are just fixing typos and such.

Reply:
We don't enforce a global license across the org, but I can ask. Thanks!